README.md

```diff
@@ -14,9 +14,9 @@ This is Ruby capstone project for Microverse program.
 - PostgresSQL
 - RSpec
 
-## Video Demo
+## Live Demo (if available)
 
-[Video Demo Link](https://drive.google.com/file/d/1Q9wd6bc0qt9-N83b4OvOnwoYHjfm0w4B/view?usp=sharing)
+[Live Demo Link](https://livedemo.com)
```

Comment:
- [ ] Kindly add a video containing a highlight about a piece of the code you find interesting or you are proud of making.
- [ ] Customize your readme file and remove unneeded titles.

Reply:
@uwadonat,  in our video I started the part where I found working on the game list interesting and also getting help from my partner.